app/components/page_heading/view.rb

```diff
@@ -0,0 +1,26 @@
+class PageHeading::View < GovukComponent::Base
+  attr_accessor :title, :errors
+  include ActiveModel
+  def initialize(title: "", errors: false)
+    @title = title
+    @errors = errors.present?
+  end
+
+  def build_page_title
```

Comment:
Could we have a spec for this component?

Reply:
Done!